ext/opentelemetry-ext-grpc/setup.cfg

```
@@ -51,3 +51,9 @@ test =
 
 [options.packages.find]
 where = src
+
+opentelemetry_instrumentor_client =
+    grpc = opentelemetry.ext.grpc:GrpcInstrumentorClient
+
+opentelemetry_instrumentor_server =
+    grpc = opentelemetry.ext.grpc:GrpcInstrumentorServer
```

the identifier must be `opentelemetry_instrumentor` for the entrypoints to be picked up by the sitecustomize. You'll also want a `[options.entry_points]` in the config

```suggestion
[options.entry_points]
opentelemetry_instrumentor =
    grpc_client = opentelemetry.ext.grpc:GrpcInstrumentorClient
    grpc_server = opentelemetry.ext.grpc:GrpcInstrumentorServer
```

---

ah my mistakw